src/Controller/UserController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace APP\Controller;
```

Comment:
Attention ici le app doit être comme ça : 
```suggestion
namespace App\Controller;
```